ui/ui-components/pages/model/[modelId]/destination/[destinationId]/edit.tsx

```diff
@@ -205,7 +205,10 @@ export default function Page(props) {
 
               <p>
                 <strong>App</strong>:{" "}
-                <Link href="/app/[id]" as={`/app/${destination.app.id}`}>
+                <Link
+                  href="/app/[id]/edit"
+                  as={`/app/${destination.app.id}/edit`}
```

Comment:
Fun fact - you don't need the "as" any more! https://github.com/grouparoo/grouparoo/pull/2475 changes all the links in the app... can you remove the 'as' and use the dynamic link as the regular href.